recipes/atef/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set package_name = "atef" %}
+{% set import_name = "atef" %}
```

Comment:
Extra, and duplicated, jinja variable obfuscates the recipe and don't really help that much.

```suggestion
```